File: Pattern Questions/Pattern3.cpp
/*Solution for
1 
1 2
1 2 3
1 2 3 4
1 2 3 4 5
*/

#include <bits/stdc++.h>
using namespace std;

void printTriangle(int n)
{
    for (int i = 1; i != n + 1; i++)
    {
        for (int j = 1; j != i + 1; j++)
        {
            cout << j << ' ';
        }
        cout << endl;
    }
}

int main()
{
    printTriangle(5);
    return 0;
}

File: Pattern Questions/Pattern3.java
public class Pattern3 {
    public static void main(String[] args) {
        
    }
    public static void printTriangle(int n){
        for(int i=1;i!=n+1;i++){
            for(int j=1;j!=i+1;j++){
                System.out.print(j);
                System.out.print(' ');
            }
            System.out.println();
        }
    }
}
